JS_2_names.js

```diff
@@ -0,0 +1,21 @@
+var arrUserNames = [];
+
+for (var i = 0; i < 5 ; i++) {
+  arrUserNames[i] = prompt('Введите имя пользователя');
+}
+
+var myName = prompt('Введите Ваше имя');
+
+var matchNames = 0;
+
+for (var i = 0; i < arrUserNames.length; i++) {
```

Comment:
Duplicated declaration of the variable `i`

Reply:
done